BuildConfigGen/dev.sh

```diff
@@ -36,13 +36,11 @@ function detect_platform_and_runtime_id ()
     fi
 }
 
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-pushd "$SCRIPT_DIR"
+SCRIPT_DIR=$1
+pushd $SCRIPT_DIR
```

Comment:
do the quotes around $SCRIPT_DIR need to be removed?

Reply:
I accidentally removed it while attempting to refactor the script. Reverted it back. Thank you.